client/data/site-profiler/metrics-dictionaries.ts

```diff
@@ -1,10 +1,5 @@
-import { BasicMetricsScored, Metrics, Scores } from './types';
-
-export const SCORES: Record< string, Scores > = {
```

Comment:
I removed this object because it was not adding any value, the type safety is achieved by the `Scores` type, not by this constant.
This object was typed as `Record < string, Scores >` so the autocompletion was not able to infer the metrics when using the `SCORES` constant (as they key is  a string!). However, when typing any value as the type `Scores`, the IDE is able to infer that we mean any of the values in the union type `good`, `bad` or `needs-improvement`

Reply:
Thanks for removing the record type here, I think just using the explicitly typed strings in the code greatly simplifies it.